cluster/manifests/metrics-server/metrics-server-vpa.yaml

```diff
@@ -10,4 +10,11 @@ spec:
       application: metrics-server
   updatePolicy:
     updateMode: "Auto"
+  {{if index .ConfigItems "prometheus_mem"}}
```

Comment:
I think there is a default value for `prometheus_mem` so the if condition is not needed.

Reply:
unfortunately not, we just set it hard in the prometheus sts if the config item doesn't exist.
https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/prometheus/statefulset.yaml#L39